recipes-domu/domu-image-android/domu-image-android.bb

```diff
@@ -1,76 +1,123 @@
-SUMMARY = "Base for Android images"
+SUMMARY = "Base for build of Android"
```

Comment:
nitpicking: "Base for Android build".

In the commit the same: "Android build" sounds better than "build of android"

Reply:
Text reworded in file and commit message.